content/igor_kolar.rst

```diff
@@ -0,0 +1,15 @@
+Igor Kolar
+##########
+
+:date: 2016-12-08 16:30
+:tags: sprinters
+:category: Sprinters
+:slug: igor-kolar
+:summary: Law / Python / Perl / Bash :)
+:image: /images/igor_kolar.jpg
```

Comment:
This has to be changed to "/images/ikolar.jpg", or the image needs to be renamed.